pkg/persistence/db.go

```diff
@@ -32,7 +32,8 @@ type Conn interface {
 	// less than 14 days ago.
 	FetchKeysForDateNumber(string, uint32, int32) ([]*pb.TemporaryExposureKey, error)
 	StoreKeys(*[32]byte, []*pb.TemporaryExposureKey) error
-	NewKeyClaim(string, string) (string, error)
+	NewKeyClaim(string, string, string) (string, error)
+	CheckHashId(string) (int64, error)
```

Comment:
Nitpick: Go style generally wants you to capitalize things like `ID` and `SHA` throughout unless it's generated code.

Reply:
Fixed in #57